docs/storage/prometheus.md

```diff
@@ -119,3 +119,4 @@ Metric name | Type | Description | Unit (where applicable) | option parameter |
 `machine_nvm_avg_power_budget_watts` | Gauge |  NVM power budget | watts | | libipmctl
 `machine_nvm_capacity` | Gauge | NVM capacity value labeled by NVM mode (memory mode or app direct mode) | bytes | | libipmctl
 `machine_thread_siblings_count` | Gauge | Number of CPU thread siblings | | cpu_topology |
+`machine_node_distance` | Gauge | Distance between NUMA node and target NUMA node | | cpu_topology |
```

Comment:
Can you put that in the proper line? I guess that metrics are listed in alphabetical order.

Reply:
Good catch! Fixing it now.

Reply:
fixed